src/plugins/intel_gpu/src/kernel_selector/kernel_selector_params.h

```diff
@@ -627,6 +627,11 @@ struct fused_operation_desc {
 struct base_params : public Params {
     virtual ~base_params() {}
 
+    enum class ArgType {
```

Comment:
Maybe reuse this enum in other operations as well? Instead StridedSliceArgType and NmsArgType

Reply:
done